src/Repository/UserMatchRepository.php

```diff
@@ -28,4 +28,24 @@ public function query(string $query) : void
         $statement = $this->entityManager->getConnection()->prepare($query);
         $statement->execute();
     }
+
+    public function findMatches($userId): array
+    {
+        $conn = $this->entityManager->getConnection();
+        $date = new \DateTime();
+        $currentDateFormatted = $date->format('Y-m-d');
+
+        $sql = '
+        SELECT um.*, user.*, FLOOR(DATEDIFF(:currentDate, user.date_of_birth)/365.25) as age
```

Comment:
turbut toks amziaus skaiciavimas yra blogai :D , bet kadangi naudojau SQL querius tai panaudojau toki

Reply:
o kam tas amziaus skaiciavimas?

Reply:
kad galetume prie card'o amziu rodyt, aisku nera butina

Reply:
tai matchai bus useriu array, o userio objektas turi birthday fielda? to ir uztenka kad atvaizduotum jo amziu twig'e